### PR TITLE
Adding Copyable feature

### PR DESCRIPTION
**Pull request by ch-hamzzyy**
#3

### CODE REVIEW

**Comment by JamesHemery** [comment]
Hi @ch-hamzzyy 

I'll test this tomorrow

File: resources/js/components/DetailField.vue [hunk 1]
```diff
@@ -1,21 +1,11 @@
 <template>
-    <PanelItem :field="field">
-        <template #value>
-            <a :href="`tel:+${unmaskedValue}`" v-if="field.value" class="no-underline text-primary">
-                {{ field.value }}
-            </a>
-            <p v-else>&mdash;</p>
-        </template>
+    <PanelItem :index="index" :field="field">
```
Comment (JamesHemery) [inline resources/js/components/DetailField.vue:2]: I think we need to copy the unmasked value (without spaces), so you can copy the nova components `js/fields/Detail/EmailField.vue`

File: resources/js/components/IndexField.vue [hunk 2]
```diff
@@ -1,14 +1,45 @@
 <template>
-    <a :href="`tel:+${unmaskedValue}`" class="no-underline text-primary"> {{ field.value }} </a>
-</template>
-
-<script>
-export default {
+    <div :class="`text-${field.textAlign}`">
```
Comment (JamesHemery) [inline resources/js/components/IndexField.vue:2]: I think we need to copy the unmasked value (without spaces), so you can copy the nova components `js/fields/Index/EmailField.vue`

**Review by JamesHemery — CHANGES_REQUESTED** [review]

**Comment by ch-hamzzyy** [comment]
I Don't think we have to do these things because of the changes, mine is working perfectly.

**Comment by JamesHemery** [comment]
> I Don't think we have to do these things because of the changes, mine is working perfectly.

When I copy phone number, I don't want to get spaces. Just change the copied value to unmasked value and I'll merge.

**Comment by JamesHemery** [comment]
Replaced by #4

**Pull request closed without merging**